rust/shared/proto/ziofa.proto

```diff
@@ -29,8 +29,10 @@ message ProcessList {
 }
 
 message Process {
-    uint32 pid = 1;
-    string package = 2;
+    int32 pid = 1;
```

Comment:
why int32 instead of uint32, can the pid be negative?

Reply:
It seems that linux itself uses i32 for pids so we should keep it as int32:

- https://internals.rust-lang.org/t/std-child-pid-type-compatibility/10473
- https://github.com/eminence/procfs/pull/250

Reply:
Yes, it matches the procfs stat.pid type.